docs/core/porting/upgrade-assistant-uwp-framework.md

```diff
@@ -0,0 +1,524 @@
+---
+title: Upgrade UWP apps to .NET 6/WinUI3/Windows App SDK
+description: Use the .NET Upgrade Assistant to upgrade an existing UWP app to .NET 6. The .NET Upgrade Assistant is a CLI tool that helps migrating an app from .NET Framework to .NET 6.
+author: adegeo
+ms.date: 05/17/2022
+---
+# Upgrade a UWP App to .NET 6 and WinUI3 with Windows App SDK capabilities using the .NET Upgrade Assistant
+
+The [.NET Upgrade Assistant](upgrade-assistant-overview.md) is a command-line tool that can assist with upgrading .NET Framework WPF apps to .NET 6. This article provides:
+
+- What to expect from the tool?
+- Things to know before starting your migration journey 
+- A demonstration of how to run the tool against a UWP app
+- Troubleshooting tips
+
+For more information on how to install the tool, see [Overview of the .NET Upgrade Assistant](upgrade-assistant-overview.md).
+
+## What to expect from the tool?
+
+We’re working on an update to the .NET Upgrade Assistant which is an open-source tool to help you automate migration of .NET Native UWP apps to .NET 6, including updating from WinUI 2 to WinUI 3. This is a command line tool that leverages .NET Upgrade assistant and will live as an extension in the same github repo [here](https://github.com/dotnet/upgrade-assistant.git). This tool aims to provide a simple migration experience by 
```

Comment:
These text editor quotes need to be changed throughout the article.
```suggestion
We're working on an update to the .NET Upgrade Assistant which is an open-source tool to help you automate migration of .NET Native UWP apps to .NET 6, including updating from WinUI 2 to WinUI 3. This is a command line tool that leverages .NET Upgrade assistant and will live as an extension in the same github repo [here](https://github.com/dotnet/upgrade-assistant.git). This tool aims to provide a simple migration experience by 
```
Also, this description seems misleading. .NET Upgrade Assistant (that should be proper capitalized as it's the name of the product) is the tool. Unless someone is running some other tool, I would say `This is a command line tool that leverages .NET Upgrade...`

Reply:
Removed the line "This is a command line tool that leverages .NET Upgrade assistant and will live as an extension in the same github repo [here](https://github.com/dotnet/upgrade-assistant.git)."